src/logger/fileWriter.c

```diff
@@ -149,13 +152,13 @@ static int write_samples_data(const LoggerMessage *msg)
         size_t count = msg->sample->channel_count;
 
         if (NULL == sample) {
-                pr_warning("Logger: null sample record\r\n");
+                pr_warning("_RCP_BASE_FILE_: null sample record\r\n");
```

Comment:
Will it?  It will be much more consistent than before (it varied between "logger", "fileLogger", "fileWriter", etc).  It will look like `fileWriter.c: Some unoriginal message`